### PR TITLE
Use `ghfs` for AnalyzeGithubContent

### PR DESCRIPTION
**Pull request by kromiii**
## Problem
Sometimes, fetching a schema file from GitHub fails because the file exceeds the file size limit in GitHub API.

https://docs.github.com/ja/rest/repos/contents?apiVersion=2022-11-28#get-contents

```
Failed to analyze schema: unsupported content encoding: none, this may occur when file size > 1 MB, if that is the case consider using DownloadContents
```

## Solution
We can avoid this by using `ghfs` (https://github.com/k1LoW/ghfs). It uses Git Database API, which has no file size limit.

## Concern
I would appriciate if you would judge whether we need to rewrite the entire code to accommodate some large files.

### CODE REVIEW

File: datasource/datasource.go [hunk 1]
```diff
@@ -167,25 +166,22 @@ func AnalyzeGitHubContent(dsn config.DSN) (*schema.Schema, error) {
...
-	if f == nil {
-		return nil, errors.Errorf("invalid dsn: %s", dsn)
-	}
-	cc, err := f.GetContent()
+	b, err := fsys.ReadFile(splitted[2])
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
-	dec := json.NewDecoder(strings.NewReader(cc))
+	dec := json.NewDecoder(strings.NewReader(string(b)))
```
Comment (k1LoW) [inline datasource/datasource.go:184]:
```suggestion
	dec := json.NewDecoder(bytes.NewReader(b))
```

**Review by k1LoW — CHANGES_REQUESTED** [review]
GREAT! Thank you!!

One minor correction, please.

**Comment by kromiii** [comment]
> GREAT! Thank you!!
> 
> One minor correction, please.

Thanks!

**Review by k1LoW — APPROVED** [review]
LGTM!

**Pull request merged**